Add entrypoints for vanity CLI

cthoyt opened the pull request:
This PR uses python entrypoints to add a vanity CLI when installing RAscore. This means that from any shell, you can now call `rascore` as a shorthand for `python -m RAscore.command_line_interface`. If you want to stylize the command line program name, that can be changed, but most users aren't expecting this. This PR also updates the README to reflect this.

cthoyt committed:
Add entrypoints for vanity CLI

cthoyt committed:
stylize CLI

cthoyt committed:
Update README.md

A-Thakkar commented:
Hi Charles, 

Thanks for this pull request. This will help make it more practical to run. 

Forgive me for my ignorance, i am struggling to get the script working through the shorthand command due to a module not found error. Could you advise as to why this might be occurring? I have reinstalled the environment and reinstalled the RAscore using setup.py.

Running the script as it was before calling the whole command works.

Any advice would be much appreciated and once it's working happy to merge the pull request.

See the error trace below:

```
Traceback (most recent call last):
  File "/home/amol/miniconda3/envs/rascore/bin/RAscore", line 33, in <module>
    sys.exit(load_entry_point('RAscore', 'console_scripts', 'RAscore')())
  File "/home/amol/miniconda3/envs/rascore/bin/RAscore", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/home/amol/miniconda3/envs/rascore/lib/python3.7/site-packages/importlib_metadata/__init__.py", line 105, in load
    module = import_module(match.group('module'))
  File "/home/amol/miniconda3/envs/rascore/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'RAScore'

```

cthoyt committed:
Fix typo

There shouldn't be a python extension, since it will actually interpret it as looking for a submodule called `py`

cthoyt commented:
@A-Thakkar I made a typo... I think it should be fixed now

A-Thakkar commented:
Closing the pull request as the modification to the entry points was added as part of resolving the other issues mentioned. Thanks alot for the modification.

cthoyt commented:
@A-Thakkar quite a shame you weren't able to merge this PR first - this leaves no record of the contribution
